ansible/files/admin_api_scripts/pg_upgrade_scripts/complete.sh

```diff
@@ -333,6 +333,9 @@ function complete_pg_upgrade {
         echo "5.1. Restarting gotrue and postgrest"
         retry 3 service gotrue restart
         retry 3 service postgrest restart
+        
+        echo "5.2. Applying netplan configuration to fix network issues"
```

Comment:
what network issues were we seeing? do we know what the underlying difference is between u20 and u24/

Reply:
I think after all this apply netplan step was not needed, as u24 does it automatically. I'll remove this and retest and thanks